solutions/ac-GCDOnBlackboard.mdx

```diff
@@ -0,0 +1,69 @@
+---
+id: ac-GCDOnBlackboard
+source: AC
+title: GCD On Blackboard
+author: Andrew Cheng
+---
+
+## Time Complexity
+$\mathcal{O}(N \log N)$ (GCD operations take $\mathcal{O}(\log N)$ time)
```

Comment:
see: https://usaco.guide/general/adding-solution?lang=cpp#steps on ordering of time complexity (and formatting)